src/main/java/com/google/sps/utils/GameUtils.java

```diff
@@ -40,4 +43,42 @@ public boolean IsValidGameName(String gameName, Entity userEntity) {
     }
     return true;
   }
+
+  /**
+  * Creates a game entity 
+  */
+  public boolean createGameEntity(String gameName, DatastoreService datastore) {
+    
+    if(gameName == "" || datastore == null){
```

Comment:
If the `datastore` is null, that should be a fatal error. Or if not, it's at least worth logging.

Reply:
I logged this with system.out as well